src/main/java/uk/gov/hmcts/cmc/claimstore/events/DocumentManagementSealedClaimHandler.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.hmcts.cmc.claimstore.events;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.claimstore.documents.CitizenSealedClaimPdfService;
+import uk.gov.hmcts.cmc.claimstore.documents.LegalSealedClaimPdfService;
+import uk.gov.hmcts.cmc.claimstore.events.claim.ClaimIssuedEvent;
+import uk.gov.hmcts.cmc.claimstore.events.solicitor.RepresentedClaimIssuedEvent;
+import uk.gov.hmcts.cmc.claimstore.services.ClaimService;
+import uk.gov.hmcts.cmc.claimstore.services.DocumentManagementService;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+
+@Component
+public class DocumentManagementSealedClaimHandler {
+    static final String PDF_EXTENSION = ".pdf";
+    static final String APPLICATION_PDF = "application/pdf";
+
+    private final DocumentManagementService documentManagementService;
+    private final LegalSealedClaimPdfService legalSealedClaimPdfService;
+    private final ClaimService claimService;
+    private final boolean documentManagementFeatureEnabled;
+    private final CitizenSealedClaimPdfService citizenSealedClaimPdfService;
+
+
+    @Autowired
+    public DocumentManagementSealedClaimHandler(
+        final DocumentManagementService documentManagementService,
+        final CitizenSealedClaimPdfService citizenSealedClaimPdfService,
+        final LegalSealedClaimPdfService legalSealedClaimPdfService,
+        final ClaimService claimService,
+        @Value("${feature_toggles.document_management}") final boolean documentManagementFeatureEnabled
```

Comment:
I'm assuming that `DocumentsService` is the entry point to the new code? You don't need this toggle in that case.